package.json

```diff
@@ -12,9 +12,9 @@
     "test:unit-hc": "VUE_APP_CONTEXT='holochain' vue-cli-service test:unit --watch=true --config=./jest.hcunit.config.js",
     "test:unit-holo": "VUE_APP_CONTEXT='holo-host' vue-cli-service test:unit --watch=true --config=./jest.holounit.config.js",
     "test-setup:happ": "nix-shell --command 'yarn fetch:dna && yarn build:happ' && exit",
-    "test-setup:holo": "yarn run clear:hc-holo && nix-shell --command 'yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc' && exit",
+    "test-setup:holo": "nix-shell --command 'yarn hc-holo:gen-agent && yarn hc-holo:run && yarn run:test-chaperone-hcc' && exit",
     "test:integration-hc": "yarn run clear:hc && yarn run clear:lair && yarn build:holochain && nix-shell --command 'VUE_APP_CONTEXT='holochain' VUE_APP_WEB_LOGS='false' VUE_APP_TEST_PORT='8888' node scripts/test.js --config=./jest.hcintegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000' && exit",
-    "test:integration-holo": "yarn test-setup:holo && yarn build:holo-hcc && node scripts/test.js --config=./jest.holointegration.config.js --watch --runInBand --detectOpenHandles --testTimeout=95000000",
```

Comment:
Why is this change made? I use `yarn test-setup:holo` plus `yarn serve:holo` for local testing and this seems like it worsens that experience>